lib/lib_rss.php

```diff
@@ -485,7 +485,7 @@ function httpGet(string $url, string $cachePath, string $type = 'html', array $a
 		// TODO: Implement HTTP 410 Gone
 	} elseif (!is_string($body) || strlen($body) === 0) {
 		$body = '';
-	} else {
```

Comment:
enforceHttpEncoding is not written to deal with JSON, but only with HTML/XML. No need to call it for JSON.